settings.yml

```diff
@@ -1,26 +1,9 @@
 custom_header_links:
   type: objects
-  default:
-    - text: "External link"
-      title: "This link will open in a new tab"
-      url: "https://meta.discourse.org"
-      view: "vdo"
-      target: "blank"
-      hide_on_scroll: "remove"
-    - text: "Most Liked"
-      title: "Posts with the most amount of likes"
-      url: "/latest/?order=op_likes"
-      view: "vdo"
-      target: "self"
-      hide_on_scroll: "keep"
-    - text: "Privacy"
-      title: "Our Privacy Policy"
-      url: "/privacy"
-      view: "vdm"
-      target: "self"
-      hide_on_scroll: "keep"
+  default: []
```

Comment:
Remove the default since we no longer need to provide examples of the format of the setting.